### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by c-schwarze** [betheluniversity/portal#49]
I know this was mentioned as something we would like to have.

We need to display notifications from various channels that require attention. For example, if the checklist has any red X's, give the user a notification of some sort.

**Comment by kelslundberg** [comment]
Here's what we decided today:

- Enrollment checklist tab will have little notification indicator in tab menu on left
- We'll use the notification icon in the top banner area 
- Homepage will have some kind of widget to get people to the enrollment checklist as well as notifications indicating how many todos are open

**Comment by c-schwarze** [comment]
The connection to the enrollment checklist is here: #50

**Comment by c-schwarze** [comment]
@tim-heck use my 'notifications' branch and hook up the enrollment checklist to this.

**Comment by c-schwarze** [comment]
This is put on hold, as we might not end up using this.

**Comment by johjacb** [comment]
i think we should add a notification bubble to 'Checklist' in the nav. i think it would really draw attention

**Comment by c-schwarze** [comment]
I agree. @tim-heck want to give this a shot when you finish up your other projects?
